token-account-based/README.md

```diff
@@ -0,0 +1,200 @@
+# Account-based token scenario
+
```

Comment:
Given that this is a new topic, should we add a discussion on the advantages of account based vs UTXO based tokens?